couchpotato/static/style/combined.min.css

```diff
@@ -984,7 +984,7 @@ input[type=text],textarea{-webkit-appearance:none}
 .page.settings .choice .select_wrapper select{min-width:0!important}
 .page.settings .renamer_to.renamer_to{-webkit-flex-flow:row wrap;-ms-flex-flow:row wrap;flex-flow:row wrap}
 .page.settings .renamer_to.renamer_to .ctrlHolder{width:100%}
-.directory_list{z-index:2;position:absolute;width:450px;margin:28px 0 20px;background:#ac0000;box-shadow:0 0 15px 2px rgba(0,0,0,.15);border-radius:3px 3px 0 0}
+.directory_list{z-index:2;position:absolute;top:60px;width:450px;margin:28px 0 20px;background:#ac0000;box-shadow:0 0 15px 2px rgba(0,0,0,.15);border-radius:3px 3px 0 0}
```

Comment:
instead of adding top:60px; you can just change the positoin from absolute to relative and the directory list will appear inside the control holder